unit_tests/test_csv_logger.py

```diff
@@ -0,0 +1,22 @@
+import os
+from scm.plams.core.csv_logger import LoggerCSV
+import pytest
+
+
+def test_csv_logger():
+    csv_logger = LoggerCSV(log_path=os.getcwd() + "/logfile.csv")
```

Comment:
Can use a temp file to avoid having to clean it up after the test:
```
with tempfile.NamedTemporaryFile() as temp_log_file:
    csv_logger = LoggerCSV(log_path=temp_log_file.name)
```